src/Core/Checkout/Cart/Command/CartMigrateCommand.php

```diff
@@ -231,9 +235,9 @@ private function sqlToRedis(InputInterface $input, OutputInterface $output): int
                     'line_item_count' => $row['line_item_count'],
                     'created_at' => $row['created_at'],
                 ]);
-            }
 
-            $this->redis->mset($values);
+                $this->redis->set($key, $value, ['EX' => $this->expireDays * 86400]);
```

Comment:
maybe redis pipeline here?

Reply:
@shyim apperently predis and phpredis don't support pipelining. Will the PR be rejected if i don't use it?

Reply:
nope then it is fine as it is

Reply:
@keulinho I change the cart.xml as suggested and added the expireDays to the tests. 

@shyim Questions on the "Missing tests" label. What type of test is missing here or was it referring to the missing 5th param?

Reply:
you could adjust the tests in a way that you assert that redis is called with the correct `EX` value e.g.
```php
        $redis->expects(static::once())
            ->method('set')
            ->with(RedisCartPersister::PREFIX . $token, $content, YourEXValue);
 ```